### DISCUSSION TITLE
Add "Scoreboard" to purge list

### DISCUSSION THREAD

**Discussion by Drc-DEV** [Zrips/CMI#1581] [Enhancement]
Add an option to clean "scoreboard.dat" on start, or an option to delete specific files maybe.
scoreboard.dat files can get huge sometimes when plugins don't clear scoreboard objectives or do not unregister scoreboards properly, so having an automatic cleanup of this particular file would be ideal. (examples: mythicmobs, citizens)

**Comment by LoneDev6** [comment]
Is it good to delete this file? I think it's the cause of high I/O and bandwidth usage
